index.html

```diff
@@ -259,8 +259,53 @@ <h3>Deployment Scenarios</h3>
         demonstrators, were described with Thing Descriptions
         that are constrained to the <a>Core Profile</a>.
       </p>
+    </section>
 
+    <!-- Use Cases -->
+  <section id="profile-use-cases">
+    <h2>Use Cases and Requirements</h2>
+    <p>
+      A set of over 30 use cases for the Web of Things were contributed by stakeholders 
+      from multiple industries for various application domains.
+      These have been published in the WoT Use Cases and Requirements document [wot-use-cases]. 
+    </p><p>
+      Based on these use cases a set of requirements have been derived which drive the development 
+      of the W3C Web of Things specification family. 
+      Several of these domains require easy integration of devices from multiple vendors, 
+      in other words, out-of-the-box interoperability. 
+      However, the descriptive approach taken by the WoT specifications generally leads 
+      to a large variety of different protocols and data formats, 
+      which can work against out-of-the box interoperability. 
+    </p><p>
+      For example, a WoT Thing Description (TD) can in theory include a description based on a 
+      networking protocol unknown to a device that wishes to connect to it.  
+      To ensure interoperability without additional customization (e.g. by writing software or 
+      performing complex setup or configuration steps), 
+      the range of such choices needs to be limited to a finite set so that a consumer 
+      of a Thing Description can be sure it will be able to interact with any possible Thing. 
+      A finite set of customization choices is also important for implementing devices with a fixed code base. 
+      Defining such constraints leads to the profile mechanism and the WoT Core profile.
+      </p>
+      <p>
+      In addition to multiple vertical use cases that will use HTTP(S) for their implementations,
+      there are two key horizontal use cases that are addressed by this profile specification.
```

Comment:
There's now only one. Might want to re-structure this.

Reply:
done